shoop/core/order_creator/_source.py

```diff
@@ -560,6 +568,8 @@ def __init__(self, source, **kwargs):
         self.require_verification = kwargs.pop("require_verification", False)
         self.accounting_identifier = kwargs.pop("accounting_identifier", "")
 
+        self.line_source = kwargs.pop("line_source", LineSource.CUSTOMER)
+
```

Comment:
`"line_source"` should be added to the `_FIELDS` list too, here https://github.com/shuup/shuup/pull/547/commits/bb95d826a2e338a733ed6f09ae78afda8b17c328#diff-fcc6ef2829a1082a78ea5a425bc5ef4dR523